worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html

```diff
@@ -76,8 +76,7 @@ <h5 class="active nav-link" style="font-weight:bolder">Create New Sprint</h5>
                 <div class="row pt-2 pb-5">
                     <div class="col-md-10"></div>
                     <div class="col-md-2">
-                        <button type="submit" class="btn" id="button">Submit</button>
-                    </div>
+                        <button type="submit" class="btn" id="button">Submit</button>                    </div>
```

Comment:
no changes done in this file, please revert it

Reply:
There was error caused due to an incorrect div tag and so @shivamgoyal7 and @sandeep-v1404 had found it was an incorrect div tag and I had to make that change

Reply:
you can revert these changes, as there is no change